stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/experimental/plugin/OfflinePlugin.kt

```diff
@@ -213,6 +263,10 @@ public class OfflinePlugin(
         )
     }
 
+    private fun requireRepos(): RepositoryFacade {
+        return repos ?: throw IllegalStateException("RepositoryFace was set too late in OfflinePlugin")
```

Comment:
Should "RepositoryFace" be "RepositoryFacade"?

Also do you think that something like the following might be more clear to the users?

```suggestion
        return repos ?: throw IllegalStateException("RepositoryFacade cannot be null")
```

This is fully up to your discretion :D

Reply:
I prefer to say that it was set too late, because this will happen in case this plugin was initialised too fast. But I'll fix the `RepositoryFacade`